tests/instruments/rohdeschwarz/test_hmp_with_device.py

```diff
@@ -28,16 +28,13 @@
 
 from pymeasure.instruments.rohdeschwarz.hmp import HMP4040
 
-pytest.skip("Only works with connected hardware", allow_module_level=True)
-
 
 @pytest.fixture(scope="session")
```

Comment:
I know that was probably not your change, but I think the scope of this fixture should be `module`, not `session` -- it's only supposed to be used in here.